### DISCUSSION TITLE
Ikke-godkendte events vises på arrangørsider

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#38] [Bugs] [answered]
Når et event sendes til godkendelse, men inden de er godkendt, så fremgår eventet for de frivillige inde under arrangørens side på portalen. 

Dette bør rettes. Ikke-godkendte events må ikke kunne vises til de frivillige, og de må ikke kunne fremsøge dem. 

Et eksempel er Kulturens Frivillige - Frederiksberg parasport, som har ansøgt om et event (framerunning camp and cup), som Kulturens Frivillige foreløbigt ikke kan godkende. 

![image](https://github.com/xooom-community/product/assets/167326849/db5814d5-f16d-40f6-80f1-54368390433b)

![image](https://github.com/xooom-community/product/assets/167326849/535179ab-bd3e-4005-828b-f03dc9275c39)

**Comment by mikkelsl** [comment, marked as answer]
Den er allerede her - du må gerne opprioritere den: https://github.com/orgs/xooom-community/projects/2/views/2?pane=issue&itemId=60857778
